src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/BuiltinUserServiceBean.java

```diff
@@ -79,6 +79,8 @@ public BuiltinUser find(Long pk) {
     public void removeUser( String userName ) {
         final BuiltinUser user = findByUserName(userName);
         if ( user != null ) {
+            // TODO: Consider adding a cascade delete instead.
+            passwordResetService.deleteResetDataByDataverseUser(user);
```

Comment:
any reason not to do cascade delete now? seems straightforward and less tech debt.

Reply:
The only reason is that I hadn't set up a cascade in a while. I'll look at some examples and see what I can do.

Reply:
Fixed in 6e61841cb.